walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -46,9 +46,13 @@ public int compare(TimeRange a, TimeRange b) {
     }
   };
 
+  /** 
+    * Compare {@code TimeRange} by their start times. 
+    */
```

Comment:
```suggestion
  /** 
   * Compare {@code TimeRange} by their start times. 
   */
```

Also add docs for the params

Reply:
the indentation of the comments look off, can you fix it for this function and for `checkRequired`

the asterisks below the first line line up with the first asterisk of `/**`. EX:
```
/** 
 *
 */
```

see other examples in this file.